project_national/panel_upgrades.yml

```diff
@@ -0,0 +1,1320 @@
+      - *7_cooling_setpoint_offset_period_none
+      - *9_heating_setpoint_offset_magnitude_0
+      - *10_heating_setpoint_offset_period_none
+    #### 1.2 Federal Minimum Heat Pump With Electric Backup, 2 options  ####
+      - *1_fed_min_ashp_ducted
+      - *2_fed_min_mshp_ductless
+    #### 2.1 Electric Water Heating, 4 options #### ##TO DO evise it to 120V HPWH
+      - *1_water_heater_fuel 
+      - *2_water_heater_fuel_eff_premium
+      - *3_water_heater_fuel_eff_standard
+      - *4_water_heater_fuel_eff_tankless
+    #### 3 Electric Resistance Cooking, 1 option  ####
+      - *electric_resistance_cooking
+    #### 4 Electric Clothes Dryer, 1 options  ####
+      - *clothes_dryer_electric 
+    #### 5 Electric Pool Heater And Hot Tub Spa Heater, 2 options  ####
+      - *1_electric_pool_heaters 
+      - *2_electric_spa_heaters
+    #### 6 Envelope, 10 options ####
+      - *1_attic_insulation_IECC_CZ1A
+      - *2_attic_insulation_IECC_CZ2A-2B-3A-3B-3C
+      - *3_attic_insulation_IECC_CZ4A-4B-4C-5A-5B-6A-6B-7A-7B
+      - *4_infiltration_30pct_reduction
+      - *5_ducts_category1
+      - *7_ducts_category3
+      - *8_ducts_category4 
+      - *9_ducts_category5 
+      - *10_Drill_and_fill
+
+  - upgrade_name: 2.6 High Efficiency and Low-voltage Electrification with Envelope  
+    options:
+    #### 1.0 Common Options For Heat Pump, 12 options ####
+      - *1_heating_fuel
+      - *2_cooling_efficiency_ducted
+      - *3_cooling_efficiency_ductless
+      - *4_100_percent_conditioned
+      - *6_cooling_setpoint_offset_magnitude_0
+      - *7_cooling_setpoint_offset_period_none
+      - *9_heating_setpoint_offset_magnitude_0
+      - *10_heating_setpoint_offset_period_none
+    #### 1.4 High-Efficiency Heat Pump With Electric Backup, 2 options  ####
+      - *1_high_eff_ashp_ducted
+      - *2_high_eff_mshp_ductless
+    #### 2.2 240V HPWH, 4 options  #### ## TO DO revise to 120V HPWH
+      - *1_water_heater_fuel_240V_hpwh
+      - *2_240V_hpwh_50_gal
+      - *3_240V_hpwh_66_gal
+      - *4_240V_hpwh_80_gal
+    #### 3 Electric Resistance Cooking, 1 option  ####
+      - *electric_resistance_cooking
+    #### 4 Electric Clothes Dryer, 1 options  ####
+      - *clothes_dryer_electric 
+    #### 5 Electric Pool Heater And Hot Tub Spa Heater, 2 options  ####
+      - *1_electric_pool_heaters 
+      - *2_electric_spa_heaters
+    #### 6 Envelope, 10 options ####
+      - *1_attic_insulation_IECC_CZ1A
+      - *2_attic_insulation_IECC_CZ2A-2B-3A-3B-3C
+      - *3_attic_insulation_IECC_CZ4A-4B-4C-5A-5B-6A-6B-7A-7B
+      - *4_infiltration_30pct_reduction
+      - *5_ducts_category1
+      - *7_ducts_category3
+      - *8_ducts_category4 
+      - *9_ducts_category5 
+      - *10_Drill_and_fill
+  
+  - upgrade_name: 2.7 BAU Electrification with Existing Fuel Backup with Envelope
+    options:
+    #### 1.0 Common Options For Heat Pump with existing backup, 8 options ####
+      - *hp_exist_bkup_heating_fuel
```

Comment:
I was thinking you can leverage apply_package_logic to further exclude shared systems, but this is okay!